testsuite/features/step_definitions/command_steps.rb

```diff
@@ -1517,17 +1519,30 @@
 
   raise 'Error getting server certificate serial!' unless result_code.zero?
 
-  command_minion = "openssl x509 --noout --text -in /etc/pki/trust/anchors/RHN-ORG-TRUSTED-SSL-CERT | grep -A1 'Serial' | grep -v 'Serial'"
   targets = %w[proxy sle_minion ssh_minion rhlike_minion deblike_minion build_host kvm_server]
   targets.each do |target|
     # get all defined minions from the environment variables and check their certificate serial
     next unless ENV.key? ENV_VAR_BY_HOST[target]
+    # Red Hat-like and Debian-like minions store their certificates in a different location
+    certificate = if target == 'rhlike_minion'
+                    '/etc/pki/ca-trust/source/anchors/RHN-ORG-TRUSTED-SSL-CERT'
+                  elsif target == 'deblike_minion'
+                    '/usr/local/share/ca-certificates/susemanager/RHN-ORG-TRUSTED-SSL-CERT.crt'
+                  else
+                    '/etc/pki/trust/anchors/RHN-ORG-TRUSTED-SSL-CERT'
+                  end
+    _, result_code_cert = get_target(target).run("test -s #{certificate}", check_errors: false)
+    raise "Error, missing certificate file for #{target}" unless result_code_cert.zero?
```

Comment:
```suggestion
    get_target(target).run("test -s #{certificate}", successcodes: [0], check_errors: true)
```

Reply:
Nice. I did not know, we can use it this way. Testing it right away.

Reply:
The test was successful. A failure looks like the following:

```bash
  Scenario: Check all new server certificates on the minions         # features/secondary/srv_rename_hostname.feature:135
      This scenario ran at: 2023-10-24 13:08:09 +0200
    When I check all certificates after renaming the server hostname # features/step_definitions/command_steps.rb:1513
      Server certificate serial: 36:8c:3f:25:b7:cd:5a:89:28:ba:93:0e:44:62:be:e0:6b:58:cf:1d
      proxy certificate serial: 36:8c:3f:25:b7:cd:5a:89:28:ba:93:0e:44:62:be:e0:6b:58:cf:1d
      sle_minion certificate serial: 36:8c:3f:25:b7:cd:5a:89:28:ba:93:0e:44:62:be:e0:6b:58:cf:1d
      ssh_minion certificate serial: 36:8c:3f:25:b7:cd:5a:89:28:ba:93:0e:44:62:be:e0:6b:58:cf:1d
      rhlike_minion certificate serial: 36:8c:3f:25:b7:cd:5a:89:28:ba:93:0e:44:62:be:e0:6b:58:cf:1d
      FAIL: test -s /etc/pki/trust/anchors/RHN-ORG-TRUSTED-SSL-CERT returned status code = 1.
      Output:
       (RuntimeError)
      ./features/support/lavanda.rb:177:in `run_local'
      ./features/support/lavanda.rb:155:in `run'
      ./features/step_definitions/command_steps.rb:1537:in `block (2 levels) in <top (required)>'
      ./features/step_definitions/command_steps.rb:1523:in `each'
      ./features/step_definitions/command_steps.rb:1523:in `/^I check all certificates after renaming the server hostname$/'
      features/secondary/srv_rename_hostname.feature:136:in `I check all certificates after renaming the server hostname'
=> /var/log/rhn/rhn_web_ui.log
```
I pushed a new commit with the changes.